NCS.DSS.EmploymentProgression/GetEmploymentProgression/Service/EmploymentProgressionGetTriggerService.cs

```diff
@@ -5,16 +5,16 @@ namespace NCS.DSS.EmploymentProgression.GetEmploymentProgression.Service
 {
     public class EmploymentProgressionGetTriggerService : IEmploymentProgressionGetTriggerService
     {
-        private readonly IDocumentDBProvider _documentDbProvider;
-        private readonly IServiceBusClient _serviceBusClient;
+        private readonly ICosmosDBProvider _documentDbProvider;
```

Comment:
Can we rename this to cosmosDBProvider please as we no longer use documentDBProvider